bsp/boards/OpenMote-CC2538/board_info.h

```diff
@@ -66,9 +83,10 @@
 #define PORT_maxRxDataPrepare               33    // 1007us (measured  84us)
 #define PORT_maxTxAckPrepare                22    //  305us (measured 219us)
 // radio speed related
-#define PORT_delayTx                        7     //  214us (measured 219us)
+#define PORT_delayTx                        12    //  214us (measured 219us)

```

Comment:
@changtengfei , I question whether setting `delayTx` is the right parameter. The idea of `delayTx` is to represent the delay between the moment the CPU say "GO, start transmitting" and the moment the SFD leaves the radio. The value of `delayTx` is hence only related to the speed of the hardware, and should have no relationship with e.g. the length of a slot.

To change the duration between a data packet and an ACK, why don't you modify `TsTxAckDelay`?